<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryTicketDialog</class>
 <widget class="QDialog" name="queryTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>queryTicket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel {
	font: 9pt &quot;幼圆&quot;;
	font-size:17px;
	color:black;
}

QPushButton {
	font: 9pt &quot;黑体&quot;;
	font-size:17px;
	color:black;
}
QDialog {
	background-color:navy;
}
QLabel {
	color:white;
}</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>42</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>42</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>catalog</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列车类别</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>42</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d</string>
   </property>
   <property name="date">
    <date>
     <year>2018</year>
     <month>6</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>42</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>departure place</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>始发地</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>42</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>destination</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目的地</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>102</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>车票信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>queryTicketDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>1093</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
